carma-messenger-ui/website/index.html

```diff
@@ -1,151 +1,136 @@
+                    </div>
+                
+                    <!--  Messenger Options Description/Help Info Modal -->
+                    <div class="modal fade" id="MessengerOptionsModal" tabindex="-1" role="dialog" aria-labelledby="MessengerOptionsModalTitle" aria-hidden="true">
+                        <div class="modal-dialog modal-dialog-centered modal-dialog modal-lg" role="document">
+                        <div class="modal-content">
+                            <div class="modal-header">
+                                <h4 class="modal-title" id="exampleModalLongTitle"  style="text-align: left;">
+                                    <strong>Messenger options description</strong>
+                                </h4>
+                                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                                    <span aria-hidden="true">&times;</span>
+                                </button>
+                            </div>
+                            <div class="modal-body" style="text-align: left;"> 
+                                <p>"Launch Platform" is checked to launch the messenger backend services including  all the ROS nodes and rosbridge connection when clicking the "Start Messenger" button.</p>    
+                                <p>"Record ROSbag" is checked to start recording and rendering the ROS logging inforamtion once the platform backend services are running.</p>                                                             
+                            </div>
+                            <div class="modal-footer">
+                            <button type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
+                            </div>
+                        </div>
+                        </div>
+                    </div>   
+
+                    <!-- Messenger Options Card -->        
+                    <div class="card" id="messenger-option-card" >
+                        <div class="card-header" style="text-align: left;padding-bottom: 0px;">
+                            Messenger Options 
+                            <span style="float: right; color: limegreen; font-size: 160%; margin:0px" id="messenger-option-description" title="Click me for more information" data-toggle="modal" data-target="#MessengerOptionsModal">
+                                <i class="fas fa-info-circle"></i>
+                            </span>
+                        </div>
+                        <div class="card-body" style="text-align: left;">
+                            <div style="margin: 5px auto;">
+                                <input type="checkbox" data-toggle="switchbutton" data-onstyle="success" data-size="lg" 
+                                    onchange="document.getElementById('rosBagEnabler').disabled = !this.checked;" 
+                                    id="cbRemoteLaunch" name="remotelaunch" checked="checked" />
```

Comment:
Please go ahead and update the name convenction from "RemoteLaunch" to "LaunchPlatform", "remotelaunch" to "launchplatform", and any other combination.

Reply:
Addressed